js/letters.js

```diff
@@ -17,6 +17,15 @@ var counter = {
 
 function countLetters(counter, sample_text){
   // FIX ME
+  if (sample_text.length === 0){
+    return counter;
+  }
+  sample_text = sample_text.toLowerCase();
+  let char = sample_text.charAt(0);
```

Comment:
You should move the `.toLowerCase` to after the `.charAt` so that you only call `.toLowerCase` on the character that you're pulling out. Otherwise, as it, it's lower casing all the characters the first time, and then needlessly calling over and over again on each recursive call.